tests/common.go

```diff
@@ -5058,6 +5058,18 @@ func collectStorkLogs(testCaseName string) {
 	collectLogsFromPods(testCaseName, storkLabel, pxNamespace, "stork")
 }
 
+// collectMongoDBLogs collects MongoDB logs and stores them using the collectLogsFromPods function
+func CollectMongoDBLogs(testCaseName string) {
+	pxbLabel := make(map[string]string)
+	pxbLabel["app.kubernetes.io/component"] = "pxc-backup-mongodb"
```

Comment:
Use the const string `mongodbStatefulset` which is provided in `backup_helper.go`

Reply:
Adding constant in mongodbStatefulset